syntax = "proto3";

message FlashApplicationCode {
  string name = 1;
  string git_version = 2;
  uint32 application_crc = 3;
  uint32 size = 4;
}

// The application code is to be transmitted in 2048-byte chunks in separate datagrams as raw data
